packages/server/src/test/java/software/uncharted/terarium/hmiserver/service/TaskServiceTest.java

```diff
@@ -106,7 +108,7 @@ public void testItCanSendGoLLMModelCardRequest() throws Exception {
 		req.setScript("gollm:model_card");
 		req.setInput(content.getBytes());
 
-		List<TaskResponse> responses = taskService.runTaskBlocking(req);
+		List<TaskResponse> responses = taskService.runTaskBlocking(req, 300);
```

Comment:
what this magic number for?

Reply:
seconds, just bumping the test to wait 5 minutes